### ISSUE TITLE
Build error when using biotope build 7.2.0 or higher

### ISSUE THREAD

**Issue by smohadjer** [biotope/biotope-build#419]
The dependency "@babel/preset-env": "7.8.4" in biotope 7.2.0 has a missing dependency, namely this one: @babel/plugin-proposal-numeric-separator. 

This causes errors for me when build runs webpack:ts task:

```
[11:34:29] Plumber found unhandled error:
Error in plugin "webpack-stream"
Message:
    ./node_modules/@biotope/element/lib/esm/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] /Users/sm/Documents/vi/roche/node_modules/@biotope/element/lib/esm/index.js: Could not find plugin "proposal-numeric-separator". Ensure there is an entry in ./available-plugins.js for it. (While processing: "/Users/sm/Documents/vi/roche/node_modules/@babel/preset-env/lib/index.js")
```

Missing plugin "proposal-numeric-separator" has been reported by many other package vendor and it seems it has been fixed in @babel/preset-env version 7.9.0 and higher. More info available here:
https://github.com/facebook/create-react-app/issues/8680

**Comment by SheepFromHeaven** [comment]
@smohadjer should be fixed by https://github.com/biotope/biotope-build/releases/tag/v7.4.1, please validate.

**Comment by smohadjer** [comment]
Thanks @SheepFromHeaven 

That error is fixed, but now I'm getting new errors for biotope's postcss-loader dependency.  For some reason this dependency doesn't get installed when I run npm install, even after deleting pakcage-lock.json.

```
Saeids-MacBook-Pro:roche sm$ npm ls postcss-loader
RocheWebFrontendFramework@5.0.36 /Users/sm/Documents/vi/roche
└── (empty)
```

If I manually install the dependency I get no errors after npm run start.

**Comment by SheepFromHeaven** [comment]
@smohadjer should be fixed with v7.4.2 :)
